src/appleseed/renderer/modeling/project/projectfileupdater.cpp

```diff
@@ -1480,6 +1480,9 @@ namespace
         {
             if (m_project.get_frame())
                 update_frame(*m_project.get_frame());
+
+            if (Scene* scene = m_project.get_scene())
```

Comment:
Please create a new revision (20), because I already updated all the test suite projects to revision 19.